packages/manager-react-components/src/hooks/datagrid/useResourcesV6.tsx

```diff
@@ -94,17 +100,28 @@ export function useResourcesV6<T = unknown>({
   const [flattenData, setFlattenData] = useState<T[]>([]);
   const { filters, addFilter, removeFilter } = useColumnFilters();
 
+  const searchableColumns = useMemo(() => {
+    return columns
+      .filter((col: ColumnDatagrid) => col.isSearchable)
+      .map((filter) => filter?.id);
```

Comment:
```suggestion
      .map((filter) => filter.id);
```
The object cannot be nullish here or it would have crash on line above